bentoml/marshal/marshal.py

```diff
@@ -310,7 +307,7 @@ def setup_routes_from_pb(self, bento_service_metadata_pb):
     async def request_dispatcher(self, request: "Request"):
         from aiohttp.web import HTTPInternalServerError, Response
 
-        with get_tracer().async_span(
+        with BentoMLContainer.tracer.get().async_span(
```

Comment:
Why don't we keep the pattern and inject tracer to `__init__`?

Reply:
Agree